modules/container/src/test/java/org/mule/runtime/container/internal/MuleClassLoaderLookupPolicyProviderTestCase.java

```diff
@@ -109,6 +110,34 @@ public void maintainsOriginalLookupStrategy() {
                hasItem(getClass().getClassLoader()));
   }
 
+  @Test
+  public void maintainsOriginalLookupStrategy2() {
```

Comment:
maintainsOriginalLookupStrategy2 -> find a name that explain what this test does without using the 2 suffix